helpers/tools.ts

```diff
@@ -188,6 +188,35 @@ For better results, you can specify the region parameter to get results from a s
       },
     ],
   },
+  {
+    display: "Azure Code Interpreter",
+    name: "azure_code_interpreter.AzureCodeInterpreterToolSpec",
+    supportedFrameworks: ["fastapi"],
+    type: ToolType.LLAMAHUB,
+    config: {
```

Comment:
@manekinekko whatever is here in `config` is generated as `config/tools.yaml` file, means this endpoint needs to be configured in that file.

This config is then passed to the constructor of the tool. 

I think it would be better if `azure_code_interpreter.AzureCodeInterpreterToolSpec` would use an envVar (see line 207 below) to configure this `pool_managment_endpoint`, as its a mandatory parameter.

Do you think we can update the `azure_code_interpreter.AzureCodeInterpreterToolSpec` tool accordingly?

Reply:
oh yes defintely, I missed that part of the config. In fact, using env vars is the approach we usually recommend. I am happy to send those fixes if you need.

Reply:
Here: https://github.com/run-llama/llama_index/pull/14732